hera_mc/mc_session.py

```diff
@@ -2062,13 +2063,15 @@ def add_correlator_config_from_corrcm(self, config_state_dict=None,
 
                     # add it to the config file table
                     self.add(config_file_obj)
+                    self.commit()
```

Comment:
Do the `commit()` calls actually perform the transactions we've set up? If so, do we need to do any error handling in case they don't stick?

Reply:
If they don't stick, the later parts of this function will fail as well, but there's not much that can be done at that point. I guess the only thing you might want to do is to remove the file from the librarian, but I'm not sure there's a good way to do that.

Reply:
Yeah, I think it's hard to do delete something from the librarian (on purpose...). I'm fine leaving this as is then.